config/webpack.config.js

```diff
@@ -40,7 +40,10 @@ module.exports = function()
 				{
 					test: /\.ts$/,
 					exclude: /node_modules/,
-					loader: 'ts'
+					loader: 'awesome-typescript-loader',
```

Comment:
Yeah, the idea was to use the same loader for both tests and distribution builds, so we know that if the tests can correctly run the generated code, the distribution builds should be fine as well.
Still, we use the normal TypeScript compiler for npm builds, so that's a difference we cannot overcome. (The 'default `ts-loader` is still different than the standalone `tsc`, so we will never get rid of that difference)